Instruments/bridge12.py

```diff
@@ -419,7 +419,16 @@ def lock_on_dip(self, ini_range=(9.81e9,9.83e9),
                     rx,freq = [self.tuning_curve_data['%gdBm_%s'%(10.0,j)] for j in ['rx','freq']]
                     rx_dBm = convert_to_power(rx)
                     rx_midpoint = (max(rx_dBm) + min(rx_dBm))/2.0
-                    #is the first rx higher than the midpoint (rx of dip/2)? If not this means we are not looking at a dip - ex. if the wg didn't switch on our rx would be a straight line and we would not have a dip
+                    # is the first rx higher than the midpoint (rx of
+                    # dip/2)? If not this means we are not looking at a
+                    # dip - ex. if the wg didn't switch on our rx would
+                    # be a straight line and we would not have a dip
+                    over_bool = rx_dBm > rx_midpoint # Contains False everywhere rx_dBm is under
+                    if not over_bool[0]:
+                        # if the dip doesn't look good,
+                        # flag an error so that we need
+                        # to try to reengage the wg.
+                        raise ValueError("Tuning Curve doesn't start over the midpoint, which doesn't make sense- check %gdBm_%s"%(10.0,'rx'))
```

Comment:
note this

Reply:
so for my understanding on try except, if we get to this raise ValueError it will move to the except block?